scripts/run_lighthouse_tests.py

```diff
@@ -79,6 +106,35 @@ def run_lighthouse_puppeteer_script():
         sys.exit(1)
 
 
+def run_webpack_compilation(source_maps=False):
```

Comment:
Just a question. You never pass in source_maps=True here, right? So do you even need the argument?

Reply:
removed code!